src/documents/activities/MatchingGame.html.md.handlebars

```diff
@@ -3,9 +3,9 @@ title: Matching Game
 layout: content
 category: Activities
 ---
+<strong>Note: This page is under construction! Additional content will be added soon.</strong>
```

Comment:
Instead of using straight HTML, use the Markdown notation. See: https://daringfireball.net/projects/markdown/syntax#em